setup.py

```diff
@@ -27,4 +37,4 @@
       tests_require=[
           'pytest'
       ]
-     )
+      )
```

Comment:
if flake8 wasn't happy and is now, then it's probably right. Could be github formatting making this hard for me to see.